### PR TITLE
[explore] Improved time filters controls

### PR DESCRIPTION
**Pull request by mistercrunch**
img

### CODE REVIEW

**Comment by timifasubaa** [comment]
LGTM

File: superset/assets/package.json [hunk 1]
```diff
@@ -9,8 +9,9 @@
   },
   "scripts": {
     "test": "mocha --require ignore-styles --compilers js:babel-core/register --require spec/helpers/browser.js --recursive spec/**/*_spec.*",
-    "cover": "babel-node node_modules/.bin/babel-istanbul cover _mocha -- --require spec/helpers/browser.js --recursive spec/**/*_spec.*",
+    "cover": "babel-node node_modules/.bin/babel-istanbul cover _mocha -- --require ignore-styles spec/helpers/browser.js --recursive spec/**/*_spec.*",
```
Comment (timifasubaa) [inline superset/assets/package.json]: Why is this necessary?

Comment (mistercrunch) [reply superset/assets/package.json]: Started failing for some reason. Seem like it had to do with the way that react-datetime was importing CSS...

File: superset/viz.py [hunk 2]
```diff
@@ -143,18 +143,14 @@ def query_obj(self):
         # potential conflicts with column that would be named `from` or `to`
         since = (
             extra_filters.get('__from') or
-            form_data.get("since") or
-            config.get("SUPERSET_DEFAULT_SINCE", "1 year ago")
+            form_data.get("since")
         )
 
         from_dttm = utils.parse_human_datetime(since)
-        now = datetime.now()
```
Comment (mistercrunch) [inline superset/viz.py]: removing this created issues where some dashboards had expression like `180 days` that would get flipped to be in the past
